backends/advanced/init.py

```
@@ -434,6 +454,8 @@ def setup_obsidian(self):
             if not neo4j_password:
                 self.console.print("[yellow][WARNING][/yellow] --enable-obsidian provided but no password")
                 neo4j_password = self.prompt_password("Neo4j password (min 8 chars)")
+
+            self.console.print(f"[green]✅[/green] Obsidian/Neo4j: enabled (configured via wizard)")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)